irony-completion.el

```diff
@@ -57,6 +57,19 @@
   :type '(repeat function)
   :group 'irony-completion)
 
+(defcustom irony-completion-accessibility-filter '(available deprecated)
```

Comment:
Just noticed, Clang calls this "AvailabilityKind", wouldn't be better to use availability in favor of accessibility?
For example, `deprecated` may fit better in availability than accessibility.